### PR TITLE
Chaining with Filter

### PR DESCRIPTION
**Pull request by code0wl**
Simplified readability 
Dist and source are now a bit smaller

### CODE REVIEW

File: src/lozad.js [hunk 1]
```diff
@@ -32,18 +32,17 @@ function markAsLoaded(element) {
   element.setAttribute('data-loaded', true)
 }
 
-const isLoaded = element => element.getAttribute('data-loaded') === 'true'
+const isLoaded = element => element.getAttribute('data-loaded')
```
Comment (thiamsantos) [inline src/lozad.js]: I think you should keep the `=== 'true'` because a `'false'` and a empty string would be evaluated as `true` too.

Comment (code0wl) [reply src/lozad.js]: @thiamsantos That is true, wondering if we can't just used a data-loaded attribute then with no value and just check if the element has a data-loaded attribute instead.

**Comment by ApoorvSaxena** [comment]
@code0wl can't see much change in the minified dist file size, can you look out for more optimisations to perform so as to have significant impact on file size.

**Comment by ApoorvSaxena** [comment]
@code0wl closing this PR for now, reopen with more enhancements

**Pull request closed without merging**